pkg/model/types.go

```diff
@@ -1,9 +1,47 @@
 package model
 
+import (
+	admiralv1 "github.com/istio-ecosystem/admiral-api/pkg/client/clientset/versioned/typed/admiral/v1"
+	"github.com/istio-ecosystem/admiral-sharding-manager/pkg/registry"
+	"sync"
+)
+
 //cluster configuration for sharding manager identity
-type Cluster struct {
+type ClusterConfig struct {
+	Name     string          `json:"name,omitempty"`
+	Locality string          `json:"locality,omitempty"`
+	Metadata ClusterMetadata `json:"metadata,omitempty"`
+}
+
+type ClusterMetadata struct {
 }
 
 //mesh workload identity configuration for cluster
-type Identity struct {
+type IdentityConfig struct {
```

Comment:
Can this be private?

Reply:
This config is used in other packages and having a a method to manage will be unnecessary, so  I think this should be Public.

Reply:
updated to be private